google/ggrc-core
# QUICK-FIX Get ACR ids dynamically via corresponding API call

# Issue description

*Access control roles ids are hardcoded for tests. So as results newly added roles required changes in selenium tests.*

# Steps to test the changes

*Add new roles into access control list that don't relate to areas covered by tests and you will see that tests start failing.*

# Solution description

*Get access control roles ids using corresponding endpoint. So it allows to get ids dynamically using returned dictionary with all access control roles.*

# Sanity checklist


# PR Review checklist
